### PR TITLE
[api][webui] Cleanup Project#release_targets_ng 

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by adrianschroeter** [comment]
I don't have much insight in this code either, but simpler is better and it is "only" used for display data in webui. So, let's merge and see.

**Comment by bgeuken** [comment]
@adrianschroeter Okay. So did you already reviewed the full pull request?

File: src/api/app/models/project.rb [hunk 1]
```diff
@@ -1645,6 +1616,27 @@ def release_targets_ng
...
+    end
+
+    package.flags.where(flag: :build, status: 'enable').each do |flag|
+      rt_key = target_mapping[flag.repo]
+      return rt_key if rt_key
+    end
+
+    nil
+  end
+  private :guess_release_target_from_package
```
Comment (ChrisBr) [inline src/api/app/models/project.rb]: I don't like 
``
private :guess_release_target_from_package
``

Don't we have already a private section in the project model?

**Comment by ChrisBr** [comment]
Is ``release_targets_ng`` tested?

**Comment by bgeuken** [comment]
I've added a test now

**Comment by bgeuken** [comment]
@ChrisBr rebased, Mind to have a second look? The one test failure is unrelated to this PR, see #1452

**Comment by ChrisBr** [comment]
LGTM

**Pull request merged**